debezium-core/src/test/java/io/debezium/transforms/RouterTest.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.transforms;
+
+import static org.fest.assertions.Assertions.assertThat;
+
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.SchemaBuilder;
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.source.SourceRecord;
+import org.junit.Test;
+
+import io.debezium.DebeziumException;
+import io.debezium.data.Envelope;
+
+/**
+ * @author Jiri Pechanec
+ */
+public class RouterTest {
+
+    private static final String LANGUAGE = "language";
+    private static final String EXPRESSION = "topic";
```

Comment:
How about naming it `topic.expression`? That'll be more in line with `topic.regex` as known from the existing logical table router.

Reply:
@gunnarmorling Renamed and docs added

Reply:
Thx!